packages/block-library/src/gallery/edit.js

```diff
@@ -686,6 +751,32 @@ export default function GalleryEdit( props ) {
 								/>
 							</ToolsPanelItem>
 						) }
+						{ aspectRatioOptions.length > 1 && (
+							<ToolsPanelItem
+								hasValue={ () =>
+									!! aspectRatio && aspectRatio !== 'auto'
+								}
+								label={ __( 'Aspect ratio' ) }
+								onDeselect={ () => setAspectRatio( 'auto' ) }
+								resetAllFilter={ () => ( {
+									aspectRatio: 'auto',
+								} ) }
```

Comment:
```suggestion
								onDeselect={ () => setAspectRatio( 'auto' ) }
```

I think `resetAllFilter` is unnecessary because we explicitly reset the aspectRatio in the restAll callback function.

Reply:
I agree